client/my-sites/plugins/plugin-eligibility/index.jsx

```diff
@@ -1,81 +0,0 @@
-import page from '@automattic/calypso-router';
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import EligibilityWarnings from 'calypso/blocks/eligibility-warnings';
-import HeaderCake from 'calypso/components/header-cake';
-import MainComponent from 'calypso/components/main';
-import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
-import { initiateThemeTransfer } from 'calypso/state/themes/actions';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
-
-class PluginEligibility extends Component {
-	static propTypes = {
-		pluginSlug: PropTypes.string,
-		siteId: PropTypes.number,
-		siteSlug: PropTypes.string,
-		translate: PropTypes.func,
-		navigateTo: PropTypes.func,
-		initiateTransfer: PropTypes.func,
-	};
-
-	getBackUrl = () => {
-		const { pluginSlug, siteSlug } = this.props;
-
-		return `/plugins/${ pluginSlug }/${ siteSlug }`;
-	};
-
-	goBack = () => this.props.navigateTo( this.getBackUrl() );
-
-	pluginTransferInitiate = () => {
-		// Use theme transfer action until we introduce generic ones that will handle both plugins and themes
-		this.props.initiateTransfer(
-			this.props.siteId,
-			null,
-			this.props.pluginSlug,
-			'',
-			'plugin_install'
-		);
-		this.goBack();
-	};
-
-	render() {
-		const { translate } = this.props;
-
-		return (
-			<MainComponent>
-				<PageViewTracker path="/plugins/:plugin/eligibility/:site" title="Plugins > Eligibility" />
-				<HeaderCake isCompact onClick={ this.goBack }>
-					{ translate( 'Install plugin' ) }
-				</HeaderCake>
-				<EligibilityWarnings
-					onProceed={ this.pluginTransferInitiate }
-					backUrl={ this.getBackUrl() }
-				/>
-			</MainComponent>
-		);
-	}
-}
-
-// It was 2:45AM, I wanted to deploy, and @dmsnell made me do it... props to @dmsnell :)
```

Comment:
@vindl, @dmsnell this is your last chance to capture this comment 😁

Reply:
Also, the time was even later, it seems 😅 :

p1481853837000448-slack-C0Q664T29

Reply:
will one of you reach out to me in a private message and help me find this link. Slack will not allow me to login in a browser and I can’t find a way to open the link in their app.

Reply:
@tyxla how much later? according to the linEd discussion the commit was marked “done” at 2:50am local time in the committer’s timezone. am I missing something?

just making sure our facts are clear 😉

Reply:
I think we're looking at the same timezone. But poor Marko was up for another hour afterward 😅

Reply:
either way, we should keep him up later. 2:50am or 3:50am is too early. there’s work to do!